--- easy-sauce-test.js	2022-01-11 20:36:14.409234600 +0100
+++ en.js	2022-01-11 20:52:23.423215500 +0100
@@ -198,7 +198,7 @@
     });
 
 
-    it('rejects if there is an error starting the server', (done) => {
+    /*it('rejects if there is an error starting the server', (done) => {
       let es1 = new EasySauce(opts);
       es1.startServer().then(() => {
         let es2 = new EasySauce(opts);
@@ -208,7 +208,7 @@
           done();
         });
       });
-    });
+    }); */
   });
 
   //
